aws-tde-pg-upgrade/05_push_results/run.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -eux
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+# Push pgbench files to the S3 bucket
+
+# Copy collected data and generated data & charts
+cp -r ${SCRIPT_DIR}/../03_execute/benchmark_data ${SCRIPT_DIR}/report-data
```

Comment:
The previous directory index needs to be updated until we have a variable that holds all the results.

```suggestion
cp -r ${SCRIPT_DIR}/../04_execute/benchmark_data ${SCRIPT_DIR}/report-data
```

Reply:
Updated